lib/v2/ymgal/radar.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    'ymgal.games': {
+        _name: '月幕 Galgame',
+        article: [
+            {
+                title: '资讯',
+                docs: 'https://docs.rsshub.app/anime.html#yue-mu-galgame',
+                source: ['/'],
+                target: '/ymgal/article/news',
+            },
+            {
+                title: '专栏',
+                docs: 'https://docs.rsshub.app/anime.html#yue-mu-galgame',
+                source: ['/'],
+                target: '/ymgal/article/co',
```

Comment:
I think this should be column according to the docs
```suggestion
                target: '/ymgal/article/column',
```